website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -37,6 +37,36 @@ The [Homebrew](https://brew.sh) package manager is required to complete the foll
    consul-k8s 0.39.0
    ```
    
+## Install a specific version of Consul K8s CLI
+
+Since Homebrew does not provide a way to install previous versions of packages, the Consul K8s CLI will need to be installed manually. Previous versions of the Consul K8s CLI could be used to install a specific version of the Consul on Kubernetes control plane. 
+
+1. Download the Consul K8s CLI using the following `curl` command. Enter the approriate version for your deployment via the `$VERSION` environment variable. 
+
+   ```shell-session
+   export VERSION=0.39.0
+   curl --location "https://releases.hashicorp.com/consul-k8s/${VERSION}/consul-k8s_${VERSION}_darwin_amd64.zip" --output consul-k8s-cli.zip
+   ```
+   
+1. Unzip the zip file ouput to extract the `consul-k8s` CLI binary. This overwrites existing files and also creates a `.consul-k8s` subdirectory in your `$HOME` folder. 
+   
+   ```shell-session
+   unzip -o consul-k8s-cli.zip -d ~/.consul-k8s
+   ```
+   
+1. Add the path to your directory. In order to persist the `$PATH` across sessions, you'll need to add this to your shellrc (i.e. shell run commands) file for the shell used by your terminal.  
+
+   ```shell-session
+   export PATH=$PATH:$HOME/.consul-k8s/
+   ```
+   
+1. (Optional) Issue the `consul-k8s version` command to verify the installation:
```

Comment:
```suggestion
1. (Optional) Issue the `consul-k8s version` command to verify the installation.
```
We prefer to end sentences with a period vs a semicolon, pr the HashiCorp writing style guide.